test/Makefile.inc

```diff
@@ -9,7 +9,12 @@ LDADD = $(PRE_LDADD) $(LIB)/libodphelper-linux.la $(LIB)/libodp-linux.la
 INCFLAGS = -I$(top_srcdir)/test \
 	-I$(top_srcdir)/platform/@with_platform@/include \
 	-I$(top_srcdir)/include \
-	-I$(top_srcdir)/helper/include
+	-I$(top_srcdir)/helper/include \
+	-I$(top_srcdir)/test \
+	-I$(top_srcdir)/test/validation/common \
+	-I$(top_builddir)/include
```

Comment:
top_builddir is alphabetically earlier than top_srcdir. Or is this ordering needed here?